pywbemtools/_utils.py

```diff
@@ -111,18 +111,18 @@ def _formatwarning(message, category, filename, lineno, line=None):
     return "{}: {}\n".format(category.__name__, message)
 
 
-def pywbemcliwarn(*args, **kwargs):
+def pywbemtools_warn(*args, **kwargs):
     """
-    Pywbemcli monkey patch for the warnings.warn function; substitutes our
+    Pywbemtools monkey patch for the warnings.warn function; substitutes our
```

Comment:
Might clarify by explaining what and why of "our own formatter". Repeated at least one other time.  Otherwise approved

Reply:
DONE - changed description to "Pywbemtools version of the warnings.warn() function, with replaced formatting."